help/generate.txt

```diff
@@ -2,4 +2,4 @@ Usage: fastify generate <FOLDER>
 
 Sets up project with `npm init -y` and
 generates a sample Fastify project
-in the <FOLDER> or in the current working directory.
+in the <FOLDER>.
```

Comment:
```suggestion
in the <FOLDER>. Specify `.` to create files in the current working directory.
```

Reply:
When using `.`, it will cause error because of this judgment:
```js
if (opts._[0] && existsSync(opts._[0])) {
    log('error', 'directory ' + opts._[0] + ' already exists')
    process.exit(1)
  }
```
I will merge this change after I improve the judgment.